INSTALL.md

```diff
@@ -103,8 +108,257 @@ docker container stop elasticsearch_test; docker container rm elasticsearch_test
 ```
 
 The following command starts a local MongoDB instance, runs the test suite, then stops and deletes the containers:
+
 ```shell
 docker run -d --name mongo_test -p 27017:27017 -d mongo:4.4.6 \
 && OPTIMADE_DATABASE_BACKEND="mongodb" py.test; \
 docker container stop mongo_test; docker container rm mongo_test
 ```
+
+## Container image
+
+### Retrieve the image
+
+The [`optimade` container image](https://github.com/Materials-Consortia/optimade-python-tools/pkgs/container/optimade) is available from the [GitHub Container registry](https://ghcr.io).
+To pull the latest version using Docker run the following:
+
+```shell
+docker pull ghcr.io/materials-consortia/optimade:latest
+```
+
+!!! note
+    The `:latest` part can be left out, as the `latest` version will be pulled by default.
+
+If you'd like to pull a specific version, this can be done by replacing `latest` in the command above with the version of choice, e.g., `0.17.1`.
+To see which versions are available, please go [here](https://github.com/Materials-Consortia/optimade-python-tools/pkgs/container/optimade/versions).
+
+### Run a container
+
+When starting a container from the image there are a few choices.
+It is possible to run both the standard OPTIMADE server implemented in this repository as well as an index meta-database from this image.
```

Comment:
```suggestion
It is possible to run either a standard OPTIMADE server, or an [index meta-database](https://github.com/Materials-Consortia/OPTIMADE/blob/master/optimade.rst#index-meta-database) server from this image.
```

Reply:
I just added the extra sentence here to underline my original point.